packages/inquirer-common/src/types.ts

```diff
@@ -54,6 +54,23 @@ export interface GuiOptions {
      * Indicate state in the left hand navigation panel in YUI
      */
     breadcrumb?: boolean | string;
+    /**
+     * Indicates how the question will be displayed in YUI
+     */
+    type?: 'label' | 'folder-browser';
```

Comment:
We have a property `guiType` for `folder-browser` on the prompt itself
See https://github.com/SAP/open-ux-tools/blob/2020de9f511a74236ff46f1fd25580f1f5232052/packages/ui5-application-inquirer/src/prompts/prompts.ts#L336

Reply:
Understood that this is added due to existing inquirers using `folder-browser`